ports/wt/portfile.cmake

```
@@ -70,12 +70,15 @@ vcpkg_cmake_configure(
         ${WT_PLATFORM_SPECIFIC_OPTIONS}
 
         -DUSE_SYSTEM_SQLITE3=ON
-        -DUSE_SYSTEM_GLEW=ON
 
         -DCMAKE_INSTALL_DIR=share
         # see https://redmine.webtoolkit.eu/issues/9646
         -DWTHTTP_CONFIGURATION=
         -DCONFIGURATION=
+        
+        "-DCMAKE_PREFIX_PATH=${CURRENT_INSTALLED_DIR}"
```

What is this meant to do?

---

Fix the following errors:
```
1> [CMake] CMake Error at F:/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package): 
1> [CMake] Could not find a package configuration file provided by "wt" with any of 
1> [CMake] the following names: 
1> [CMake] 
1> [CMake] wtConfig.cmake 
1> [CMake] wt-config.cmake 
1> [CMake] 
1> [CMake] Add the installation prefix of "wt" to CMAKE_PREFIX_PATH or set "wt_DIR" to 
1> [CMake] a directory containing one of the above files. If "wt" provides a separate 
1> [CMake] development package or SDK, be sure it has been installed. 
1> [CMake] Call Stack (most recent call first): 
1> [CMake] CMakeUsage/CMakeLists.txt:17 (find_package)
```

https://github.com/emweb/wt/blob/4.10.4/CMakeLists.txt#L142-L148

---

So you don't want to and need to override the vcpkg-provided CMAKE_PREFIX_PATH, but set input variable USERLIB_PREFIX instead.

https://github.com/emweb/wt/blob/c10dbd54e3151fa48851f30aff9ad0a582f1a7b6/CMakeLists.txt#L156-L157

```suggestion
        "-DUSERLIB_PREFIX=${CURRENT_INSTALLED_DIR}"
```

And maybe this still isn't safe for debug builds.